content/md/first-steps/manage-families.md

```diff
@@ -1,112 +1,102 @@
 ---
 id: manage-your-families
 themes: catalog-settings
-title: Manage your **families**
+title: Manage your **families and variant families**
 popular: false
 ee-only: false
 related: what-is-a-family
 ---
 
-# List families
+:::info
```

Comment:
Maybe to remove, as we already have a link to this article at the end in the `Related articles` section?

Reply:
Hum, do you want me to remove the List Families part? or put back the text we had before and remove the info part? (I think we should remove the 1st part: List families)